atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -278,8 +277,8 @@ def run(self):
             base_image_labels = {}
         else:
             try:
-                # OSBS2 TBD
-                config = imageutil.base_image_inspect()[INSPECT_CONFIG]
+                # OSBS2 TBD: decide if we need to inspect a specific arch
```

Comment:
atm we are generating auto labels which are arch specific (but they don't depend on base image ones)

and adding aliases for old labels (which shouldn't be arch specific)

but any of this could change in the future, we we probably should get arch specific just to be sure

Reply:
Yup, changed comment to suggest inspecting correct arch